pkg/tools/bash_tool.go

```diff
@@ -24,16 +24,25 @@ import (
 	"strings"
 
 	"github.com/GoogleCloudPlatform/kubectl-ai/gollm"
+	"k8s.io/klog/v2"
 )
 
+var bashBin string
+
 func init() {
+	// Find the bash executable path using exec.LookPath.
+	// On some systems (like NixOS), executables might not be in standard locations like /bin/bash.
+	var err error
+	bashBin, err = exec.LookPath("bash")
```

Comment:
Can we move this lookup closer to the command execution. Keeping package initialization side-effect free is preferable and also in some cases (simple question/answer) bash is never even invoked.

Also, kubectl execution is also done using bash, so lets do it in a way it works for both.

Reply:
Fixed by adding a new function `lookupBashBin`, please take a look, thanks!